File: Resueltos_por_competidor/Gabriella_Castro/OnlyPluses.java
/*
 * Autor: Gabriella Castro
 * Problema: Only Pluses
 * Juez online: VJudge
 * Veredicto: Accepted
 * URL: https://vjudge.net/problem/CodeForces-1992A
 */

import java.util.Scanner;

public class Main{

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		int t = sc.nextInt();

		for (int i = 0; i < t; i++) {

			int a = sc.nextInt(), b = sc.nextInt(), c = sc.nextInt();

			for (int j = 1; j <= 5; j++) {

				if (a <= b && a <= c) {
					a = a + 1;
				} else if (b <= c) {
					b = b + 1;
				} else {
					c = c + 1;
				}
			}
			
			System.out.println(a*b*c);

		}

	}

}

File: Resueltos_por_competidor/Alejandra_Carvajal/OnlyPluses.cpp
/* Autor: Alejandra Carvajal
* Problema: Only pluses
* Juez online: Codeforces
* Veredicto: Accepted
* URL: https://codeforces.com/problemset/problem/1992/A
*/
#include <bits/stdc++.h>
using namespace std;

int main() {
	int t; cin >>t;
    for(int i=0; i<t;i++){
        int a,b,c; cin >>a >>b >>c;
        vector<int> banana;
        banana.push_back(a);
        banana.push_back(b);
        banana.push_back(c);

        int sum = 5;
        while(sum--){
            sort(banana.begin(), banana.end());
            banana[0]++;
        }
        int m = 1;
        for (int i: banana){
            m *= i;
        }
        cout << m << endl;

    }

    return 0;
}
